src/utils/payloadStringify.test.js

```diff
@@ -333,6 +333,17 @@ describe('payloadStringify', () => {
     ]);
   });
 
+  test('shallowMask -> requestBody -> LUMIGO_SECRET_MASKING_REGEX -> bad regex', () => {
+    utils.setDebug();
+    process.env[LUMIGO_SECRET_MASKING_REGEX] = '["a(a"]';
+    expect(shallowMask('requestBody', { a: 'b', aa: 'bla' })).toEqual({ a: 'b', aa: 'bla' });
+    expect(ConsoleWritesForTesting.getLogs()).toEqual([
+      expect.objectContaining({
+        msg: '#LUMIGO# - WARNING - "Failed to parse the given masking regex"',
```

Comment:
This does not reflect the message you've added:
```js
invalidMaskingRegexWarning(`Error processing regex: ${x} - ${e.message}`)
```
are we testing the right thing here?

Reply:
updated, 
should be `invalidMaskingRegexWarning(e);`